model/project.go

```diff
@@ -1493,8 +1513,22 @@ func (p *Project) BuildProjectTVPairs(patchDoc *patch.Patch, alias string) {
 // mapping of the build variant to the tasks that will run on that build
 // variant. If includeDeps is set, it will also resolve task dependencies.
 func (p *Project) ResolvePatchVTs(patchDoc *patch.Patch, requester, alias string, includeDeps bool) (resolvedBVs []string, resolvedTasks []string, vts []patch.VariantTasks) {
-	bvs := patchDoc.BuildVariants
-	tasks := patchDoc.Tasks
+	var bvs, bvTags, tasks, taskTags []string
+	for _, bv := range patchDoc.BuildVariants {
+		if strings.HasPrefix(bv, ".") {
```

Comment:
Do we maybe want to add a comment explaining why a "." prefix is the deciding factor here?
(I only know because I see the tests and the cli example in the description. If I was scanning the code quickly without more context, I wouldn't know.)